Util/Xamarin.Kotlin.BindingSupport/source/Xamarin.Kotlin.BindingSupport.targets

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="_KotlinBindingFixups" AfterTargets="ExportJarToXml"
+          Inputs="$(ApiOutputFile);@(EmbeddedJar);@(InputJar);@(EmbeddedReferenceJar);@(ReferenceJar);@(_AdditionalJavaLibraryReferences)"
+          Outputs="$(IntermediateOutputPath)\KotlinBindingSupport.generated.xml">
+
+    <!-- public properties -->
+    <PropertyGroup>
+      <KotlinBindingSupportJar Condition="'$(KotlinBindingSupportJar)' == ''">$(MSBuildThisFileDirectory)binding-support-1.0.jar</KotlinBindingSupportJar>
+      <KotlinBindingSupportExtraArgs Condition="'$(KotlinBindingSupportExtraArgs)' == ''"></KotlinBindingSupportExtraArgs>
+      <KotlinBindingSupportOutputPath Condition="'$(KotlinBindingSupportOutputPath)' == ''">$(IntermediateOutputPath)\KotlinBindingSupport.generated.xml</KotlinBindingSupportOutputPath>
+    </PropertyGroup>
+
+    <!-- private properties -->
+    <PropertyGroup>
+      <_KotlinBindingJavaPath Condition="'$(_AndroidXJavaPath)' == '' and '$(OS)' == 'Unix'">$(JavaSdkDirectory)\bin\java</_KotlinBindingJavaPath>
```

Comment:
Isn't this exposed already somewhere for us by the android targets?

Reply:
The JavaSdkDirectory is, I am just adding the jave.exe. Most of the time, the tasks just take the folder and then use a bunch of things, such as the java.exe. I just need the java.exe so just get it.